src/components/comment/comment.stories.mdx

```diff
@@ -28,22 +61,100 @@ At minimum, a single comment consists of:
 This information may be passed to the component as a `comment` object matching the structure of a [Timber comment](https://timber.github.io/docs/reference/timber-comment/).
 
 <Canvas>
-  <Story name="Single">{template({ comment: makeComment() })}</Story>
+  <Story
+    name="Single"
+    parameters={{
+      docs: {
+        source: {
+          code: makeTwigInclude(
+            '@cloudfour/components/comment-form/comment-form.twig',
```

Comment:
Does this need to be updated? I'm not sure. 🤔 
```suggestion
            '@cloudfour/components/comment/comment.twig',
```

Reply:
Oh yeah, good catch :+1: